### ISSUE TITLE
Command line option for running tests by substring match?

### ISSUE THREAD

**Issue by joshski** [mochajs/mocha#1570]
It can be annoying to express test names as regular expressions in order to `--grep` specific tests, especially when you are copying and pasting the name of a failing test in your terminal.

For example, on todays mocha repository:

```
> mocha --grep "grep()"
  ․․․․․․․․․

  9 passing (69ms)
```

```
> mocha --grep "grep\(\)"
  ․․․․․

  5 passing (61ms)
```

Is there an easy way of avoiding this issue? If not, perhaps the mocha command line would be a little friendlier with an option to avoid escaping:

```
> mocha --matching "grep()"
  ․․․․․

  5 passing (61ms)
```

I implemented a small change to achieve this: https://github.com/joshski/mocha/commit/e0643208d5ba363c2d238f4afb6ff684bfdedfa4

Please say if you'd like a PR.

**Comment by refractalize** [comment]
+1

**Comment by a8m** [comment]
Did you try `--fgrep` ?

**Comment by danielstjules** [comment]
Exactly what @a8m mentioned :) Give fgrep a shot. It's available as of 2.2.0, though I'd suggest using 2.2.1.

```javascript
$ ./bin/mocha --fgrep 'grep()'

  ․․․․․

  5 passing (60ms)
```

**Comment by joshski** [comment]
Great, thanks!
